pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ast/ExprBetween.java

```diff
@@ -44,6 +44,12 @@ public <R> R accept(AstVisitor<R> visitor) {
     public final Expr lowerBound;
     public final Expr upperBound;
 
+    public boolean forTimestampParam;
```

Comment:
```suggestion
    public boolean forTimestampParam = false;
```

Reply:
Default value of boolean values is false. So, the above two lines have the same behavior.